pkg/system/utimes_unsupported.go

```diff
@@ -1,10 +1,10 @@
-// +build !linux,!freebsd,!darwin
+// +build !linux,!freebsd
 
 package system
 
 import "syscall"
 
-// LUtimesNano is not supported on platforms other than linux, freebsd and darwin.
+// LUtimesNano is not supported on platforms other than linux and freebsd.
```

Comment:
nit: perhaps change to `LUtimesNano is only supported on linux and freebsd.` (it's a bit hard to read currently)